athena-cloudera-hive/src/main/java/com/amazonaws/athena/connectors/cloudera/HiveMetadataHandler.java

```diff
@@ -225,6 +242,38 @@ public GetSplitsResponse doGetSplits(BlockAllocator blockAllocator, GetSplitsReq
         }
         return new GetSplitsResponse(getSplitsRequest.getCatalogName(), splits, null);
     }
+
+    /**
+     * Overridden this method to describe the types of capabilities supported by a data source
+     * @param allocator Tool for creating and managing Apache Arrow Blocks.
+     * @param request Provides details about the catalog being used.
+     * @return A GetDataSourceCapabilitiesResponse object which returns a map of supported capabilities
+     */
+    @Override
+    public GetDataSourceCapabilitiesResponse doGetDataSourceCapabilities(BlockAllocator allocator, GetDataSourceCapabilitiesRequest request)
+    {
+        Set<StandardFunctions> unsupportedFunctions = ImmutableSet.of(NULLIF_FUNCTION_NAME, IS_DISTINCT_FROM_OPERATOR_FUNCTION_NAME, ARRAY_CONSTRUCTOR_FUNCTION_NAME);
```

Comment:
If we aren't supporting ARRAY_CONSTRUCTOR_FUNCTION_NAME, then why did we implement athena-cloudera-hive/src/main/java/com/amazonaws/athena/connectors/cloudera/HiveFederationExpressionParser.java?